Dockerfile

```diff
@@ -1,18 +1,18 @@
-# Use Linux-Alpine image
 FROM amazoncorretto:17.0.12-alpine
 
-RUN apk -U upgrade
+RUN apk update && apk -U upgrade && rm -rf /var/cache/apk/*
 
-ARG JAR_LIB_FILE=./app/build/libs/app-all.jar
+RUN adduser -S myLowPrivilegeUser
+USER myLowPrivilegeUser
 
-# Create directory and switch to it
-WORKDIR /app
+# Set the workdir to a location that the running application can write to
+# which is in the myLowPrivilegeUser home folder because we are running as that user instead of root.
+WORKDIR /home/myLowPrivilegeUser/app/
 
-# Add application JAR to created folder
-COPY ${JAR_LIB_FILE} app.jar
+# Copy the jar file into /usr/local/bin/ because it seemingly needs to go to a location that any user can access.
+# If we put the jar file into the myLowPrivilegeUser's home directly, the container fails to run in Azure.
+COPY --chown=myLowPrivilegeUser ./app/build/libs/app-all.jar /usr/local/bin/app.jar
 
-# Run the api
```

Comment:
Same as previous comment, including line 17.

Reply:
I thought it was obvious what was happening.  Perhaps not?  Do you think otherwise, and I should put it back?

Reply:
I prefer having step by step because I am unfamiliar and it helps having plain context. I was mainly curious since they were there before

Reply:
Sure, I'll add them back.

Reply:
Done.